src/App.tsx

```diff
@@ -1,203 +1,126 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import Header from './components/Header';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/Errors';
+import { Status } from './types/Status';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorType, setErrorType] = useState<ErrorType | null>(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
Consider renaming `isLoading` to improve readability 
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig